### PR TITLE
Remove unused imports

### LINKED ISSUE #103: Some imports are never used?
We import

https://github.com/COSIMA/regional-mom6/blob/3e9844286db015e17940bbbacc2c35f7ddc82e23/regional_mom6/regional_mom6.py#L2

https://github.com/COSIMA/regional-mom6/blob/3e9844286db015e17940bbbacc2c35f7ddc82e23/regional_mom6/regional_mom6.py#L5

https://github.com/COSIMA/regional-mom6/blob/3e9844286db015e17940bbbacc2c35f7ddc82e23/regional_mom6/regional_mom6.py#L12

but I don't see where we use them.

Are they actually needed?

### PR DESCRIPTION
**Pull request by navidcy**
I think some of the imports are not used so I remove them here and see what happens.

Closes #103

### CODE REVIEW

**Comment by navidcy** [comment]
hm... test pass so they were probably no needed?

File: regional_mom6/regional_mom6.py [hunk 1]
```diff
@@ -1,21 +1,19 @@
 import numpy as np
-from itertools import cycle
 from pathlib import Path
 import dask.array as da
-import dask.bag as db
 import numpy as np
```
Comment (angus-g) [inline regional_mom6/regional_mom6.py:4]: We also import numpy twice!

**Review by angus-g — APPROVED** [review]
Yep, good catch! I've just dropped out the duplicated numpy import too.

File: regional_mom6/regional_mom6.py [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
-from itertools import cycle
```
Comment (navidcy) [inline regional_mom6/regional_mom6.py:2]: @angus-g, is `itertools` an unnecessary dependency of the package?

Comment (navidcy) [reply regional_mom6/regional_mom6.py:2]: hm... I guess `itertools` is part of Python so we are all good!

Comment (angus-g) [reply regional_mom6/regional_mom6.py:2]: No, it's a Python built-in.

Comment (navidcy) [reply regional_mom6/regional_mom6.py:2]: I'm learning... :)

**Pull request merged**